Dockerfile

```diff
@@ -15,13 +15,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM debian:buster
+FROM mpioperator/openmpi-builder AS builder
```

Comment:
Do we need to tie this to a specific version of the image?

Reply:
I'd prefer `latest` at this point in the program. We can lock it down closer to release

Reply:
Inverted my logic: Fixed the mpioperator images to the 0.3.0 tag; if something comes into latest that we need we can jump to it then (or, more likely, the next published tag)